### ISSUE TITLE
prepareCriterion in CriteriaProcessor escaping already escaped date

### ISSUE THREAD

**Issue by johnywith1n** [balderdashy/waterline-sequel#20]
In [criteriaProcessor.js](https://github.com/balderdashy/waterline-sequel/blob/master/sequel/lib/criteriaProcessor.js), the function ```prepareCriterion``` doesn't check if a value is already an escaped date when its processing NOT clauses. 

This is creating SQL statements like:

```
SELECT *  WHERE ((`message`.`createdAt` <> "\"2015-01-21 20:09:47\"" ) OR (`message`.`id` < 75 )) AND `message`.`createdAt` <= "2015-01-21 20:09:47" ORDER BY `message`.`createdAt` DESC, `message`.`id` ASC LIMIT 1
```
specifically:

```
"\"2015-01-21 20:09:47\""
```

I'm not sure if this also affects the cases below it, but they also don't check for already escaped dates.

**Comment by johnywith1n** [comment]
Was this check left out for any particular reason?

**Comment by CWyrtzen** [comment]
Ninja'd~

**Comment by sailsbot** [comment]
Thanks for posting, @johnywith1n.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
